### PR TITLE
Negative indices for start_line

### PR DESCRIPTION
**Pull request by saimn**
Following PR #26, modify `_get_line_index` to handle negative line index, and add a test for this.

### CODE REVIEW

File: asciitable/basic.py [hunk 1]
```diff
@@ -107,8 +107,11 @@ def write(self, lines):
         lines.append(self.write_comment + self.splitter.join([x.name for x in self.cols]))
 
 class CommentedHeader(core.BaseReader):
-    """Read a file where the column names are given in a line that begins with the
-    header comment character.  The default delimiter is the <space> character.::
+    """Read a file where the column names are given in a line that begins with
+    the header comment character. `header_start` can be used to specific the
```
Comment (taldcroft) [inline asciitable/basic.py]: Minor typo "to specific" should be "to specify".

**Comment by taldcroft** [comment]
Apart from the docstring typo this looks good.  If you just fix that and push then I'll go ahead and merge this.  The issue with the latex units has been there for a while, I'll plan to fix that and put things together for a minor release.

**Comment by saimn** [comment]
The typo is fixed, thanks for the review.
Another question I have concern astropy: if I understand well astropy.io.ascii is the same codebase as asciitable ? So you report the changes between the two projects ?

**Comment by taldcroft** [comment]
Thanks for the contribution @saimn.  I also fixed the latex test issue and that is in master.  Right now there is a transition period between `asciitable` and `astropy.io.ascii`.  I had originally planned to maintain the two code bases in parallel but this ends up requiring a fair bit of extra work, mostly in order to maintain backward compatibility.  So moving forward I am focusing development effort on the astropy version which will break compatibility with `asciitable`.

If you're interested you could apply the basically same patch to astropy.io.ascii and submit a PR there (astropy/astropy).  Maybe I should have suggested this right away, but astropy is still in alpha and not yet widely advertised.  BTW, if you do end up patching astropy.io.ascii it would be easier if you don't do all the whitespace cleanup.  This makes the patch cleaner and easier to see what's happening.  I've been meaning to clean up the code for whitespace and PEP8 (and will before the next release) but not quite yet.  :-)

In any case this is a good addition and will get into io.ascii one way or another.

**Comment by saimn** [comment]
I understand for the extra work it represents. I made the change in `asciitable` because I use `atpy` to convert ascii files to FITS. But I suppose that atpy will also move to use `astropy.io.ascii`, so I will submit this PR to astropy (without the whitespace cleanup, sorry for that, I forget about this as it is automated in my emacs config).

**Comment by astrofrog** [comment]
Just to confirm - ATpy will indeed be moving to use `astropy.io.ascii` in the next release!

**Pull request merged**